src/Engine/GraphicsHandler/I_GraphicsHandler.h

```diff
@@ -8,19 +8,191 @@
 class I_GraphicsHandler {
 public:
     virtual ~I_GraphicsHandler() {}
+    /**
+     * Used to intiate the GraphicsHandler
+     * Loads the vector of <code>graphic_paths</code> which are file paths
+     * to <code>GraphicsHandler::game_graphics_</code>, which is a map of std::string to SDL_Textures*
+     * @param path is the relative (recommended) or absolute path to graphics assets that will 
+     *  be loaded when init is called. This initial load avoids having to load assets while the
+     *  level has started, and can be done when "loading" the level
+     * @return returns <code>void</code>. However, <code>GraphicsHandler::internal_load_image()</code> calls 
+     * <code>load_image</code> which _can_ potentially call <code>LOG(FATAL)</code>
+     * and exit the program when an incorrect path is called.
+     * \note
+     * We have no real error handling here. If an invalid path is given, it means an object may not have
+     * an asset, so calling exiting the program is kind of appropriate. However, maybe one day, we should
```

Comment:
what do you mean an object may not have an asset?

Also, removing the "calling" and the "kind of"

Reply:
What I meant is that if the given path is invalid, then essentially, the asset would not be loaded. Therefore the object (missile.h) for example, would not have a loaded asset (the missile.png)

Reply:
I just realized that this isn't the case on init. Init will just cache things ahead of time. The objects will try to grab from a path anyways.